client/state/plans/get-interval-type-for-term.js

```diff
@@ -0,0 +1,17 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+
+import { TERM_ANNUALLY, TERM_BIENNIALLY, TERM_MONTHLY } from 'lib/plans/constants';
+
+export default function getIntervalTypeFromTerm( term ) {
+	return (
```

Comment:
The `return` could be ditched if we use an arrow function.

Reply:
I actually rather like "classic" `function`s for top-level functions like these 🙂 

Let's leave the style up to @adamziel here 👍